### DISCUSSION TITLE
Filtering specific warnings

### DISCUSSION THREAD

**Discussion by GCaggianese** [mrcjkb/haskell-tools.nvim#353] [Q&A] [answered]
Hello,

I am trying to filter out some specific warnings in my Haskell project. I've set up a `.hlint.yaml` file in the root directory of my project, specifying certain warnings to ignore. Running `hlint .` from the terminal honors the `.hlint.yaml` configuration and does not show the ignored warnings. However, when I work within Neovim, these warnings continue to appear despite the `.hlint.yaml` setup.

I have looked through the documentation but have not found a clear solution for ensuring Neovim and Haskell Tools respect the `.hlint.yaml` settings.

Could you please provide guidance on how to make Neovim with Haskell Tools  recognize and apply the `.hlint.yaml` settings?

Thank you for your assistance.

**Comment by mrcjkb** [comment, marked as answer]
Hey :wave:

linting is done by haskell-language-server's `hls-hlint-plugin`.
Check out [this comment](https://github.com/haskell/haskell-language-server/issues/3127#issuecomment-1952647197) on a related haskell-language-server issue - maybe it'll have the info you need :smile:

**Reply by GCaggianese** [reply]
Hey!

Really thank you for your answer, I managed to make it work! 😅

Also thank you for the amazing plugin!
